Dockerfile

```diff
@@ -28,14 +27,18 @@ RUN git checkout $(git describe --abbrev=0 --tags $VERSION)
 RUN { [ $(ls /src/modules/ | wc -l) -gt 0 ] && cp -r /src/modules/* /inspircd-src/src/modules/ || echo "No modules overwritten/added by repository"; }
 RUN echo $EXTRASMODULES | xargs --no-run-if-empty ./modulemanager install
 
-RUN ./configure $CONFIGUREARGS --prefix /inspircd --uid 10000 --gid 10000
+RUN ./configure $CONFIGUREARGS --uid 10000 --gid 10000
+RUN ./configure --prefix /inspircd --uid 10000 --gid 10000
 RUN make -j`getconf _NPROCESSORS_ONLN` install
 
 ## Wipe out vanilla config; entrypoint.sh will handle repopulating it at runtime
 RUN rm -rf /inspircd/conf/*
 
 # Stage 1: Create optimized runtime container
 FROM alpine:3.9
+
+ARG RUN_DEPENDENCIES=
```

Comment:
Any specific reason why moving this one down here? Cache optimization?

Reply:
https://docs.docker.com/engine/reference/builder/#scope

```
An ARG instruction goes out of scope at the end of the build stage where it was defined. To use an arg in multiple stages, each stage must include the ARG instruction.
```

If you define RUN_DEPENDENCIES in the first stage, is not defined in the second stage

Reply:
Oh I didn't notice we declare it in the state, thought we declare it above. Good catch then!